namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-uat/resources/elasticache.tf

```diff
@@ -0,0 +1,35 @@
+module "elasticache" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=6.0.0"
+  vpc_name               = var.vpc_name
+  team_name              = var.application
+  namespace              = var.namespace
+  business-unit          = var.business_unit
+  application            = var.application
+  is-production          = var.is_production
+  environment-name       = var.environment
+  infrastructure-support = var.infrastructure_support
+  node_type              = "cache.t4g.micro"
+
+  # Consider setting the engine to a recent version (defaults to 5.0.6)
+  engine_version = "6.x"
```

Comment:
do we not need to specify redis

looking at: 
https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-production/resources/elasticache.tf#L19-L20